lib/python/qmk/keyboard.py

```diff
@@ -9,6 +9,25 @@
 from qmk.c_parse import parse_config_h_file
 from qmk.makefile import parse_rules_mk_file
 
+BOX_DRAWING_CHARACTERS = {  # noqa: this reads better across multiple lines
+    "unicode": {
+        "tl": "┌",
+        "tr": "┐",
+        "bl": "└",
+        "br": "┘",
+        "v": "│",
+        "h": "─"
```

Comment:
A trailing comma will indicate to yapf that we want multiple lines here.

```suggestion
        "h": "─",
```